test/specs/lobbyAdmitParticipant.js

```diff
@@ -0,0 +1,85 @@
+const LobbyNotification = require("../page-objects/notifications/LobbyNotification");
+const Toolbox = require("../page-objects/Toolbox");
+const SecurityDialog = require("../page-objects/SecurityDialog");
+import {
+    BASE_URL,
+    DEFAULT_CONFIG,
+    ENTER_KEY,
+    FIRST_PARTICIPANT,
+    SECOND_PARTICIPANT
+} from "../helpers/constants"
+import createFirefoxSession from "../helpers/firefoxSession";
+import createChromeSession from "../helpers/chromeSession";
+import openParticipantsPane from "../helpers/openParticipantsPane";
+
+describe('Activate lobby and admit participant', () => {
+    let roomName;
+    let capabilities;
+    let Guest1;
+    it('should open jitsi-meet app and enable lobby by first participant', async () => {
+        capabilities = await browser.requestedCapabilities;
```

Comment:
What is this?

Reply:
Removed. Just tested something related to browser capabilities.